queries/markdown/injections.scm

```diff
@@ -1,8 +1,11 @@
 (fenced_code_block
   (info_string
-    (language) @language)
-  (#not-match? @language "elm")
-  (code_fence_content) @content (#exclude_children! @content))
+    (language) @_lang)
+  (#not-match? @_lang "elm") ; prevent segfault when using elm parser
```

Comment:
Sorry for not mentioning, but is there any other parser that causes segfault here? Aside from this, I agree with the change

Reply:
Don't think so, Nim might if we add it